src/instructlab/sdg/configs/knowledge/data_recipe/default_recipe.yaml

```diff
@@ -0,0 +1 @@
+datasets: []
```

Comment:
Let's just remove this empty file and just create an empty Recipe() in the code?

Reply:
Downstream users would need to change the contents of the package to change this recipe file anyway, so I don't think it's a problem to also change 1 line of code

If we really want to support downstream recipes, this is not how to do it - and we should file an issue for that requirement

I really don't want to leave stuff like this in place - someone upstream is quite likely to remove it, not understanding the downstream consequences. And if downstream no longer needs it, they're very unlikely to notify upstream of this.

That's how we end up with cruft that people are afraid to touch. Not a huge deal in this case, but broken windows and all that

Reply:
Downstream we would be unlikely to put any default data file in the code package. Instead, we would install it separately as part of building the application image.

Reply:
I'd really rather not assume we're going to manage patches downstream for packages where we have enough influence to ensure the design doesn't require it. We have enough work doing that for packages where we have no influence over the maintainers.

Reply:
See https://github.com/instructlab/sdg/pull/203#issuecomment-2250444499